gremlin-dotnet/test/Gremlin.Net.UnitTest/Structure/IO/GraphSON/GraphSONReaderTests.cs

```diff
@@ -39,7 +39,13 @@ public class GraphSONReaderTests
         private GraphSONReader CreateStandardGraphSONReader()
         {
             return new GraphSONReader();
-            }
+        }
+
+        [Fact]
+        public void Assert_loaded_Newtonsoft_Json_version()
+        {
+            Assert.Equal(new Version(11, 0, 0, 0), typeof(JToken).Assembly.GetName().Version);
```

Comment:
Sorry but I completely missed this test in my last review. I thought that you just added the test to find out what was causing the other test to fail, but I don't think that we should add a test like that as we would have to change it every time we update Newtonsoft.Json.

Reply:
I can remove it, but since it is still unclear what forces that 9.0.0 reference, we might actually just leave it in and update it on a Newtonsoft.Json-update, it's easily done. It might be helpful in looking for the root cause for that 9.0.0 dependency and for future mishaps where the wrong version loads.

Reply:
Ok, good point. But then I would suggest that you add a comment that explains why version 11 is needed and maybe also change the test name to be more in line with the other tests. How about something like _NewtonsoftJsonVersionShouldSupportReallyBigIntegers_?